### ISSUE TITLE
Support @relay(plural: true) directive

### ISSUE THREAD

**Issue by chollier** [guymers/babel-plugin-flow-relay-query#2]
Currently there is no way to generate fragments like that :

``` javascript
    answers: () => Relay.QL`
      fragment on Answer @relay(plural: true) {
        __id
        location { lat lng }
      }
    `,
```

I tried playing with the source code but couldn't really get anything close to supporting it. I was thinking of an API like this :

``` javascript
    answers: () => generateFragmentFromProps("Answer", { plural: true }),
```

**Comment by guymers** [comment]
I've updated the maker function to now take an object given everything is optional:

``` javascript
type FragmentOptions = {
  name?: string,
  directives?: {
    [name: string]: { [arg: string]: boolean|number|string }
  }
}

function generateFragmentFromProps(options?: FragmentOptions): Function {}
```

So to add a relay directive with plural = true you would

``` javascript
answers: () => generateFragmentFromProps({
  name: "Answer",
  directives: {
    relay: { plural: true }
  }
})
```

Is that too generic? Would it be simpler to only accept a map of relay directive arguments?

**Comment by chollier** [comment]
This definitely make sense! Thanks :) would you mind pushing a new release?

**Comment by guymers** [comment]
Sure, released 0.10.0
